README.md

```diff
@@ -75,6 +75,11 @@ roddy.sh run projectConfigurationName@analysisName patientId \
 
 TBD
 
+
+### Cross-Species Contaminations
+
+In coding regions, the expected proportion of synonymous mutations compared to the total number of mutations should be low. On the other hand, a high proportion of synonymous mutations may be an indicator of cross-species contamination. Any value above 0.5 (50% of mutations are synonymous) is a clear indicator of contamination. Anything below 0.35 is considered to be OK. Values in the range of 0.35-0.5 are unclear. 
```

Comment:
indicator for / indicative of
0.5 (at least 50% of...
ist there any reference for mentioned thresholds?